src/components/DataView/DataViewColumnDrawerContent.tsx

```diff
@@ -41,6 +41,15 @@ const StyledWrapper = styled.div`
 		display: flex;
 		justify-content: space-between;
 		padding-left: 15px;
+
+		& > span {

```

Comment:
This is also affecting the buttons inside the .right > .item as they also have spans inside of them. Please change this to only affect the fist span that is a child of the .right > .item. You can do this with & > span:first-child